main.go

```diff
@@ -116,11 +118,33 @@ func main() {
 		prerelease,
 	)
 
+	viper.AddConfigPath(".")
```

Comment:
Support environment variables 
```suggestion
	viper.AutomaticEnv()
	viper.SetEnvPrefix("svu")
	viper.AddConfigPath(".")
```